.github/workflows/trigger_prerelease.yml

```diff
@@ -0,0 +1,18 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# For more details about how to release follow https://github.com/newrelic/coreint-automation/blob/main/docs/release_runbook.md
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 15 * * 1"
+
+jobs:
+  prerelease:
+    uses: newrelic/coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
```

Comment:
We may want to update this to ignore the `tests` folder when the `exclude-dirs` parameter is exposed.